tests/css_scss/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2048,6 +2084,41 @@ label {
   @include margin-bottom-1\\/3;
 }
 
+@forward "src/list";
+@forward "src/list" as list- *;
+@forward "src/list" as list- *;
+@forward "src/list" hide list-reset, $horizontal-list-gap;
+@forward "src/list" hide list-reset, $horizontal-list-gap,
+  $horizontal-list-gap-foo, $horizontal-list-gap-bar;
+
+@use "src/code";
+@use "src/code" as c;
+@use "src/code" as c;
+@use "src/code" as *;
+@use "library" with ($black: #222, $border-radius: 0.1rem);
+@use "library" with
+  ($black: #222, $border-radius: 0.1rem, $left: 10px, $right: 400px);
+@use "library" with
+  (
+    $black: #222,
+    $border-radius: 0.1rem,
+    $left: 10px,
+    $right: 400px,
+    $box-shadow: 0 0.5rem 1rem rgba($black, 0.15)
+  );
+@use "veryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLong"
+  with
+  (
+    $black: #222,
+    $border-radius: 0.1rem,
+    $left: 10px,
+    $right: 400px,
+    $box-shadow: 0 0.5rem 1rem rgba($black, 0.15),
+    0 0.5rem 1rem rgba($black, 0.15),
+    0 0.5rem 1rem rgba($black, 0.15),
+    0 0.5rem 1rem rgba($black, 0.15)
+  );
```

Comment:
Ugly, we should format that:
```scss
@use 'veryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVery' with (
  $black: #222,
  $border-radius: 0.1rem
);
```

Reply:
@evilebottnawi any ideas on what type the AST node should be. I tried converting the `with` to a function with a nested `paren_group` but that formats it as
```scss
@use 'veryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVery' with(
  $black: #222,
  $border-radius: 0.1rem
);
```

Reply:
No ideas :disappointed: just point on bad output